pkg/controllers/cloud/networkpolicy.go

```diff
@@ -254,41 +253,42 @@ func vpcsFromGroupMembers(members []antreanetworking.GroupMember, r *NetworkPoli
 			continue
 		}
 		var cloudRsc securitygroup.CloudResource
-		readAnnotations := false
-		if kind == converter.GetExternalEntityLabelKind(&cloud.VirtualMachine{}) {
+		readOwnerLabels := false
+		if kind == converter.GetExternalEntityLabelKind(&runtimev1alpha1.VirtualMachine{}) {
 			cloudRsc.Type = securitygroup.CloudResourceTypeVM
-			readAnnotations = true
+			readOwnerLabels = true
 		} else {
 			r.Log.Error(fmt.Errorf(""), "invalid cloud resource type received", "kind", kind)
 		}
-		if readAnnotations {
-			ownerAnnotations, ownerCloudProvider, err := getOwnerProperties(e, r)
+		if readOwnerLabels {
+			ownerVM, err := getOwnerVM(e, r)
 			if err != nil {
 				r.Log.Error(err, "externalEntity owner not found", "key", key, "kind", kind)
 				namespacedName := &types.NamespacedName{Namespace: m.ExternalEntity.Namespace, Name: m.ExternalEntity.Name}
 				notFoundMember = append(notFoundMember, namespacedName)
 				continue
 			}
-			vpc, ok := ownerAnnotations[cloudcommon.AnnotationCloudAssignedVPCIDKey]
+			//cloudAccountID, ok := ownerVM.Labels[config.LabelCloudNamespacedAccountName]
+			cloudAccountName, ok := ownerVM.Labels[config.LabelCloudAccountName]
 			if !ok {
-				r.Log.Error(fmt.Errorf(""), "vpc annotation not found in ExternalEntity owner", "key", key, "kind", kind)
-				continue
-			}
-			cloudRsc.Vpc = vpc
-			vpcs[vpc] = append(vpcs[vpc], &cloudRsc)
-			cloudAssignedID, ok := ownerAnnotations[cloudcommon.AnnotationCloudAssignedIDKey]
-			if !ok {
-				r.Log.Error(fmt.Errorf(""), "cloud assigned ID annotation not found in ExternalEntity owner", "key", key, "kind", kind)
+				r.Log.Error(fmt.Errorf(""), "cloud account ID annotation not found in ExternalEntity owner", "key", key, "kind", kind)
```

Comment:
this is cloud account fetched from label

Reply:
updated the error message.